Userland/Libraries/LibVideo/PlaybackManager.h

```diff
@@ -145,12 +156,13 @@ class PlaybackManager {
 
     static DecoderErrorOr<NonnullOwnPtr<PlaybackManager>> create(NonnullOwnPtr<Demuxer> demuxer);
 
-    void start_timer(int milliseconds);
     void timer_callback();
+    // This must be called with m_demuxer_mutex locked!
     Optional<Time> seek_demuxer_to_most_recent_keyframe(Time timestamp, Optional<Time> earliest_available_sample = OptionalNone());
```

Comment:
Feels like a fixme, we should be clever enough to come up with an API that requires that lock in order to call it 🤔

Reply:
I think Jelle brought this up too, and I would like to do that, but I didn't find a reasonable way to implement that check unfortunately. I'm open to solutions, though.